crates/astria-proto/proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -51,10 +51,10 @@ message TransferAction {
 // `SequenceAction` represents a transaction destined for another
 // chain, ordered by the sequencer.
 //
-// It contains the chain ID of the destination chain, and the
+// It contains the rollup ID of the destination chain, and the
 // opaque transaction data.
 message SequenceAction {
-    bytes chain_id = 1;
+    bytes rollup_id = 1;
```

Comment:
Renamed this field to be in line with renaming chain ID -> rollup ID.

This is not a breaking protobuf change (field names don't matter)